Ruins/src/main/java/atomicstryker/ruins/common/RuinTemplateRule.java

```diff
@@ -607,7 +607,8 @@ else if (dataString.startsWith("ChestGenHook:"))
         {
             String[] s = dataString.split(":");
             int targetCount = s.length > 2 ? Integer.valueOf(s[s.length-1].split("-")[0]) : 0;
-            addChestGenChest(world, random, x, y, z, s[1], targetCount, blocknum, rotate);
+            String lootTable = s.length > 3 ? s[1]+s[2] : s[1];

```

Comment:
You need to restore the colon between the namespace and path here, so s[1] + s[2] should be s[1] + ":" + s[2].

Reply:
Never mind...it looks like I wasn't looking your the latest code.